blocks/results-list-block/features/results-list/results/result-item.jsx

```diff
@@ -1,9 +1,28 @@
 import React from "react";
+import { useEditableContent } from "fusion:content";
+import { useComponentContext } from "fusion:context";
+import getProperties from "fusion:properties";
+import getTranslatedPhrases from "fusion:intl";
+import { localizeDateTime } from "@wpmedia/engine-theme-sdk";
 
-import ArticleDate from "@wpmedia/date-block";
-import { Image } from "@wpmedia/engine-theme-sdk";
 import { extractResizedParams, extractImageFromStory } from "@wpmedia/resizer-image-block";
```

Comment:
We should not be relying on the resizer-image-block in v2

Reply:
@matthewroach Does this block support video like promo blocks or just an image?

Reply:
The function `extractImageFromStory` is available in Arc Themes Components - https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-2.0.3/src/utils/get-image-from-ans/index.js 
We do not need the function `extraResizedParams` in V2 as we should be using Image Resizer v2 which allows us to put image sizes directly on the image component and not have to do them ahead of time